testsuite/features/secondary/minkvm_guests.feature

```diff
@@ -342,6 +342,95 @@ Feature: Be able to manage KVM virtual machines via the GUI
     Then I wait until I do not see "test-net1" text
     And I should not see a "test-net1" virtual network on "kvm_server"
 
+# Start provisioning scenarios
+
+# This scenario needs the 15-SP2 base system channel to be synchronized
+@long_test
+@virthost_kvm
+@scc_credentials
+  Scenario: Create auto installation distribution
+    Given I am on the Systems page
```

Comment:
I'm not sure we need this step `Given I am on the Systems page` to navigate to Distributions.

Reply:
it failed to work without it IIRC

Reply:
Probably what you need is something like Given I am authorized as admin admin

Reply:
Worked, changed